stable/admin/README.md

```diff
@@ -1,32 +1,16 @@
 # NuoDB Admin Helm Chart
```

Comment:
this chart requires special privileges (the SCC) in OCP. If you are going to mention stuff in the database chart, it should also be mentioned here

Reply:
I'm not sure what "stuff" is. Please be specific or we can hold additional updates for the next PR following this one.

Reply:
All privileges needed are mentioned on the stable page.